client/client_pool/src/concord_client_pool.cpp

```diff
@@ -154,9 +154,9 @@ SubmitResult ConcordClientPool::SendRequest(std::vector<uint8_t> &&request,
     LOG_WARN(logger_, "Cannot allocate client for" << KVLOG(correlation_id));
     if (callback) {
       if (serving_candidates == 0 && !clients_.empty()) {
-        callback(bftEngine::SendResult{SubmitResult::ClientUnavailable});
+        callback(bftEngine::SendResult{OperationResult::NOT_READY});
       } else {
-        callback(bftEngine::SendResult{SubmitResult::Overloaded});
```

Comment:
SubmitResult::Overloaded means that we don't have a free client to serve the request. Please leave it as is.